IFC4x3/Properties/d/CargoDeadWeight_2vVxFsGGb3QOAIX9UEyhga/Documentation.md

```diff
@@ -1 +0,0 @@
-Weight of (bulk) cargo carried
```

Comment:
As Above is a replacement for 'DeadWeightTonnage' should not be deleted.

Reply:
Refer above, this file was duplicated in d folder